meta-lmp-bsp/conf/machine/include/lmp-machine-custom.inc

```diff
@@ -347,6 +347,7 @@ OSTREE_SPLIT_BOOT:mx8mm-nxp-bsp = "${@bb.utils.contains('DISTRO_FEATURES', 'luks
 OSTREE_DEPLOY_USR_OSTREE_BOOT:mx8mm-nxp-bsp = "${@bb.utils.contains('DISTRO_FEATURES', 'luks', '1', '0', d)}"
 WKS_FILE:sota:mx8mm-nxp-bsp = "${@bb.utils.contains('DISTRO_FEATURES', 'luks', 'sdimage-imx8-spl-split-boot-sota.wks.in', 'sdimage-imx8-spl-sota.wks.in', d)}"
 ## iMX8MM EVK
+DISTRO_FEATURES:remove:imx8mm-lpddr4-evk = "vulkan"
```

Comment:
I'd move it to the common imx8mm part and use the upstream classifier `mx8mm-nxp-bsp` - we can add support of other imx8mm boards in the future, so such generalization simplifies steps for adding such support.

Reply:
I agree with the more generic use of `mx8mm-nxp-bsp` machine override and the move to common imx8mm

Reply:
Fixed, thanks for point that.